src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml

```diff
@@ -27,19 +27,52 @@
     <Canvas Background="Transparent"
             Name="RootLayout" >
 
-        <Polygon x:Name="TooltipPointer"
-                 Visibility="{Binding ShowPopupPointer, Converter={StaticResource BooleanToVisibilityConverter}}"
-                 Points="{Binding TooltipPointerPoints}" 
-                 Style="{StaticResource PoupPolygonPointerStyle}" />
-
-        <Path x:Name="PopupPathRectangle" 
+        <Path x:Name="PopupPathRectangle"   
               Style="{StaticResource PoupPathRectangleStyle}">
             <Path.Data>
                 <RectangleGeometry x:Name="BackgroundRectangle">
                 </RectangleGeometry>
             </Path.Data>
+            <!--This effect will show a 4px shadow of 20% of tranparency with a angle of 135 grades-->
+            <Path.Effect>
+                <DropShadowEffect Opacity="0.2" 
+                                  Color="Black"
+                                  ShadowDepth="4"
+                                  BlurRadius="4"
+                                  Direction="135"/>
+            </Path.Effect>
         </Path>
 
+        <Path x:Name="PopupPathRectangleShadow"   
+              Style="{StaticResource PoupPathRectangleStyle}">
+            <Path.Data>
+                <RectangleGeometry Rect="{Binding ElementName=BackgroundRectangle, Path=Rect}">
+                </RectangleGeometry>
+            </Path.Data>
+            <!--This effect will show a 4px shadow of 20% of tranparency with a angle of 315 grades-->
+            <Path.Effect>
+                <DropShadowEffect Opacity="0.2" 
+                                  Color="Black"
+                                  ShadowDepth="4"
+                                  BlurRadius="4"
+                                  Direction="315"/>
+            </Path.Effect>
+        </Path>
+
+
+        <Polygon x:Name="TooltipPointer"
```

Comment:
Some comments to this block would be appreciated!

Reply:
@QilongTang I added extra comments in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12048/commits/c84d82b2d65d9ce4ea1dcf90649974be6f4618cd